src/main/java/core/basesyntax/CreationOfFigure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class CreationOfFigure extends FigureSupplier {
```

Comment:
Class with method `main` shouldn't inherit any logic. FigureSupplier should be method-level field. Please, folow the rule [is-a, has-a](https://stackoverflow.com/questions/2218937/has-a-is-a-terminology-in-object-oriented-language)
```suggestion
public class Main {
```